### ISSUE TITLE
Draw route with animation like UBER app.

### ISSUE THREAD

**Issue by VivekVithlani** [heremaps/here-ios-sdk-examples#138]
Hello,

Any way to draw a route with animation like UBER App. ?

Thank you,
Vivek Vithalani.

**Comment by dashchak** [comment]
Hi @VivekVithlani, 
unfortunately there is no way to animate the route drawings in HERE SDK.

**Comment by moment101** [comment]
Maybe use a timer to add short lines one by one. and then remove them to achieve the animation effect you want.

**Comment by VivekVithlani** [comment]
Yes, I also plan for a timer for draw route, but in here map just one method for assign a route on map, which is ```- (BOOL)addMapObject:(nonnull NMAMapObject *)object``` in this method we can just pass route as map object, we can't put lines one by one.
